prov/efa/src/efa_fabric.c

```diff
@@ -481,6 +477,28 @@ static int efa_alloc_fid_nic(struct fi_info *fi, struct efa_context *ctx,
 	return ret;
 }
 
+static int efa_get_gdr_support(char *device_name)
+{
+	char *gdr_path = NULL;
+	char gdr_support = '0';
+	int ret;
+
+	ret = asprintf(&gdr_path, "class/infiniband/%s/device/gdr", device_name);
+	if (ret < 0) {
+		EFA_INFO_ERRNO(FI_LOG_FABRIC, "asprintf", ret);
```

Comment:
Can you put a more descriptive message here? Looks like this macro doesn't have a line number/function name.

Reply:
It does:

```
libfabric:49004:efa:fabric:efa_get_gdr_support():486<info> asprintf: <snip>
```
But I made it better anyway.

Reply:
Oh, I see that now in `FI_LOG`, thanks for the update regardless.